lambdas/appointment_email.py

```diff
@@ -91,7 +100,7 @@ def send_cal_email(supp, supp_email, student, student_email, appt_time, duration
     #     attendees.append(x)
     #s = supp.split('@')
     
-    organizer = "ORGANIZER;CN=ReachOUT:mailto:umassreachout.dev"+CRLF+" @gmail.com"
+    organizer = "ORGANIZER;CN=ReachOUT:MAILTO:umassreachout.dev@gmail.com"
```

Comment:
What was the reason behind this?

Reply:
Is there a constants file or something for static data like this?

Reply:
We are going to be shifting these to environment variables in lambda later.

Reply:
Shifted to environment variables